Dockerfile

```diff
@@ -11,9 +11,10 @@ RUN test -n "${IBC_GO_VERSION}"
 
 # Grab the static library and copy it to location that will be found by the linker flag `-lwasmvm_muslc`.
 # TODO: nice to have: a script to download version matching the wasmvm version in go.mod.
-ADD https://github.com/CosmWasm/wasmvm/releases/download/v1.2.4/libwasmvm_muslc.x86_64.a /lib/libwasmvm_muslc.x86_64.a
-RUN sha256sum /lib/libwasmvm_muslc.x86_64.a | grep ce3d892377d2523cf563e01120cb1436f9343f80be952c93f66aa94f5737b661
-RUN cp /lib/libwasmvm_muslc.x86_64.a /lib/libwasmvm_muslc.a
+ADD https://github.com/CosmWasm/wasmvm/releases/download/v1.4.0/libwasmvm_muslc.aarch64.a /lib/libwasmvm_muslc.aarch64.a
+ADD https://github.com/CosmWasm/wasmvm/releases/download/v1.4.0/libwasmvm_muslc.x86_64.a /lib/libwasmvm_muslc.x86_64.a
+RUN sha256sum /lib/libwasmvm_muslc.aarch64.a | grep 2a72c7062e3c791792b3dab781c815c9a76083a7997ce6f9f2799aaf577f3c25
+RUN sha256sum /lib/libwasmvm_muslc.x86_64.a | grep 8ea2e3b5fae83e671da2bb51115adc88591045953f509955ec38dc02ea5a7b94
```

Comment:
looks like we need to update the `modules/light-clients/08-wasm/Dockerfile` instead. But there's no harm actually just updating both for now.

Reply:
hopefully in the call today we'll be able to clean up the dep we currently have on 08-wasm in ibc-go. when that's done, a couple of things will be able to be removed including these changes to the Dockerfile

Reply:
we also wouldn't need to get the aarch64 lib? We should only be running actual tests on x86_64 machine.